src/v0/sources/customerio/mapping.json

```diff
@@ -25,5 +25,6 @@
   "data.device_platform": "properties.devicePlatform",
   "data.recipients": "properties.recipients",
   "data.event_type": "properties.type",
-  "data.newsletter_id": "properties.newsletterId"
+  "data.newsletter_id": "properties.newsletterId",
+  "data.transactional_message_id": "properties.transactionalMessageId"
```

Comment:
Is this a new field included in your documentation? I am following [this](https://customer.io/docs/api/webhooks/#operation/reportingWebhook) link but couldn't find this field. Can you point me to the correct documentation if there is any?

Reply:
A link to the webhook attribute documentation can be found in the PR description:
https://www.customer.io/docs/webhooks/#webhook-attributes

There you can see the `transactional_message_id` attribute.